### DISCUSSION TITLE
[M-Z] (13 Giugno 2022) Esercizio 9

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#473] [Esami & Esoneri M-Z]
### Traccia

![image](https://github.com/user-attachments/assets/cf06ad5a-65a7-4dd9-836d-fc6e26480962)

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20220613.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20220613.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/cc0f2f4c-67a9-4ffb-9cbd-7cd39316d938)
